doc/source/conf.py

```diff
@@ -132,7 +134,8 @@
 # make rst_epilog a variable, so you can add other epilog parts to it
 rst_epilog = ""
 # Read link all targets from file
-with Path.open("links.rst") as file:
+path_to_links_rst = Path(__file__).parent / "links.rst"
+with path_to_links_rst.open() as file:
     rst_epilog += file.read()
```

Comment:
Again, if we are only reading for appending text, it is better to avoid the context and use the `read_text` method.